query-engine/connector-test-kit/src/test/scala/queries/OrderBySpec.scala

```diff
@@ -79,6 +81,21 @@ class OrderBySpec extends FlatSpec with Matchers with ApiSpecBase {
     result.toString should be("""{"data":{"needsTiebreakers":[{"order":1},{"order":2},{"order":3},{"order":4},{"order":5},{"order":6},{"order":7}]}}""")
   }
 
+  "The order when giving an order by ASC that is unique" should "be by Id ascending and therefore oldest first" in {
```

Comment:
I don't understand the logic. If we are ordering by a unique field, isn't it a case where we don't need a tie breaker?

Reply:
Yep, I should change the test naming. The uniques are created in the same order as the id, so  its the same.